Formula/aircrack-ng.rb

```diff
@@ -16,19 +19,27 @@ class AircrackNg < Formula
     sha256 "02efed81e48c8f70bbd1d3051e84b25815fcceb7166cb79d472f9552a4708ae2" => :mavericks
   end
 
+  devel do
+    url "https://download.aircrack-ng.org/aircrack-ng-1.2-rc4.tar.gz"
+    sha256 "d93ac16aade5b4d37ab8cdf6ce4b855835096ccf83deb65ffdeff6d666eaff36"
+    patch do
+      # The url will be replaced with one on formula-patch once PR on formula-patch has merged
+      url "https://github.com/equal-l2/formula-patches/raw/aircrack-ng/aircrack-ng/aircrack-ng-devel.patch"
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!